### ISSUE TITLE
Trailing blank in user e-mail causes homebridge crash

### ISSUE THREAD

**Issue by nyjklein** [pschroeder89/homebridge-levoit-humidifiers#62]
**Describe The Bug:**
Trailing blank in user e-mail causes homebridge crash and continuous restart cycle

**To Reproduce:**
Insert a Trailing blank in user e-mail via either the GUI or the json.

**Expected behavior:**
No crash

**Logs:**

```
[02/01/2024, 12:05:07] [Levoit Humidifiers] Discovering devices...
[02/01/2024, 12:05:07] Error: The user is not logged in!
    at /var/lib/homebridge/node_modules/homebridge-levoit-humidifiers/src/api/VeSync.ts:278:15
    at /var/lib/homebridge/node_modules/homebridge-levoit-humidifiers/node_modules/async-lock/lib/index.js:171:12
    at AsyncLock._promiseTry (/var/lib/homebridge/node_modules/homebridge-levoit-humidifiers/node_modules/async-lock/lib/index.js:306:31)
    at exec (/var/lib/homebridge/node_modules/homebridge-levoit-humidifiers/node_modules/async-lock/lib/index.js:170:9)
    at AsyncLock.acquire (/var/lib/homebridge/node_modules/homebridge-levoit-humidifiers/node_modules/async-lock/lib/index.js:189:3)
    at VeSync.getDevices (/var/lib/homebridge/node_modules/homebridge-levoit-humidifiers/src/api/VeSync.ts:276:17)
    at Platform.discoverDevices (/var/lib/homebridge/node_modules/homebridge-levoit-humidifiers/src/platform.ts:80:39)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
```

**Plugin Config:**

```json
Show your Homebridge config.json here, remove any sensitive information.
```

**Screenshots:**

**Environment:**

* **v1.11.0**:
* **v1,7,0**: 
* **v20.10.0**: 
* **NPM Version**: 
* **Debian GNU/Linux Bullseye (11)**:

**Comment by pschroeder89** [comment]
Not sure I understand the use case here. Why are you entering the email with a trailing space?

**Comment by nyjklein** [comment]
It was an accident. I used an autofill from my keyboard which added the trailing blank. Not really a use case. But it really shouldn't send HB into a crash/restart loop!

**Comment by pschroeder89** [comment]
Fixed in 1.12.0
<img width="690" alt="image" src="https://github.com/pschroeder89/homebridge-levoit-humidifiers/assets/10761540/f375580a-ea4f-4930-9771-2eb713370fdf">
